iSeg/ImageInformationDialogs.h

```diff
@@ -107,40 +107,37 @@ class RotationDialog : public QDialog
 	RotationDialog(SlicesHandler* hand3D, QWidget* parent = 0,
 				   const char* name = 0, Qt::WindowFlags wFlags = 0);
 	~RotationDialog();
-	void return_direction_cosines(float dc[6]);
+	void get_rotation(float r[3][3]);
```

Comment:
no matrix class?

Reply:
iseg does not use much of a base class library, e.g. for math. you are right that some/plenty of code could be safer/easier to extend & maintain if we added classes for matrices,etc.